artifacts/install/apiserver-template.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: namespace-reservation-server
+parameters:
+- name: IMAGE
+  value: namespace-reservation-server:latest
+- name: NAMESPACE
+  value: openshift-namespace-reservation
+- name: LOGLEVEL
+  value: "0"
```

Comment:
Is 0 a good idea here? Ran into this recently, a pod with no logs always makes me wonder.

Reply:
> Is 0 a good idea here? Ran into this recently, a pod with no logs always makes me wonder.

0 is more verbose than info and setting it in a template gives the power to bump it easily.